Dockerfile

```diff
@@ -0,0 +1,21 @@
+FROM debian:bookworm-slim AS base
+
+RUN apt update -qy
+RUN apt install -qy librocksdb-dev
+
+FROM base as build
+
+RUN apt install -qy git cargo clang cmake
+
+WORKDIR /build
+COPY . .
```

Comment:
Should add a `.dockerignore` file with `Dockerfile` included so that changes to Dockerfile don't invalidate the cache of this COPY step.

Reply:
Done